src/parquet_format.rs

```diff
@@ -36,14 +36,14 @@ use thrift::server::TProcessor;
 /// would handle this.
 #[derive(Copy, Clone, Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
 pub enum Type {
-  BOOLEAN = 0,
```

Comment:
Just curious: is there a way to avoid these changes with Thrift 0.12.0? It will be great if we can avoid a backward incompatible change caused by these.

Reply:
It doesn't seem like it unfortunately.